migration_scripts/README.md

```diff
@@ -17,9 +17,293 @@ python3 migration_scripts/*.py ; #run the migration script
+```
+
+#### [src/data/moves_info.h](../src/data/moves_info.h)
+```diff
+    [MOVE_POUND] =
+    {
+        .name = COMPOUND_STRING("Pound"),
+        .description = COMPOUND_STRING(
+            "Pounds the foe with\n"
+            "forelegs or tail."),
+        .effect = EFFECT_HIT,
+        .power = 40,
+        .type = TYPE_NORMAL,
+        .accuracy = 100,
+        .pp = 35,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .category = DAMAGE_CATEGORY_PHYSICAL,
+        .makesContact = TRUE,
+        .ignoresKingsRock = B_UPDATED_MOVE_FLAGS == GEN_4,
+        .contestEffect = CONTEST_EFFECT_HIGHLY_APPEALING,
+        .contestCategory = CONTEST_CATEGORY_TOUGH,
+        .contestComboStarterId = COMBO_STARTER_POUND,
+-        .contestComboMoves = {0}
++        .contestComboMoves = {0},
++        .battleAnimScript = Move_POUND,
+    },
+```
+
+### Item Icons
+
+* Filepath [`migration_scripts/1.9/convert_item_icons.py`](1.9/convert_item_icons.py)
+* Introduced in [Get rid of gItemIconTable #4579](https://github.com/rh-hideout/pokeemerald-expansion/pull/4579)
+
+Moves all information from `gItemIconTable` to `gItemsInfo`.
+
+#### [src/data/item_icon_table.h](..src/data/item_icon_table.h)
+```diff
+-    [ITEM_POKE_BALL] = {gItemIcon_PokeBall, gItemIconPalette_PokeBall},
+```
+
+#### [src/data/items.h](..src/data/items.h)
+```diff
+    [ITEM_POKE_BALL] =
+    {
+        .name = _("Poké Ball"),
+        .price = 200,
+        .description = COMPOUND_STRING(
+            "A tool used for\n"
+            "catching wild\n"
+            "Pokémon."),
+        .pocket = POCKET_POKE_BALLS,
+        .type = ITEM_USE_BAG_MENU,
+        .battleUsage = EFFECT_ITEM_THROW_BALL,
+        .secondaryId = ITEM_POKE_BALL - FIRST_BALL,
++        .iconSprite = gItemIcon_PokeBall,
++        .iconPalette = gItemIconPalette_PokeBall,
+    },
+```
+
+### Partner & Trainer Parties
+
+* Filepaths [`migration_scripts/1.9/convert_trainer_parties.py`](1.9/convert_trainer_parties.py) and [`migration_scripts/1.9/convert_partner_parties.py`](1.9/convert_trainer_parties.py)
+* Introduced in [Competitive-formatted parties #3545](https://github.com/rh-hideout/pokeemerald-expansion/pull/3545) and [Make trainerproc compatible with partners #4421](https://github.com/rh-hideout/pokeemerald-expansion/pull/4421)
+
+Converts trainer data (for `convert_trainer_parties.py`) and partner data (for `convert_partner_data.py`) to the new `.party` format. Not necessary if `COMPETITIVE_PARTY_SYNTAX` is turned off.
+
+#### [src/data/trainer_parties.h](..src/data/trainer_parties.h)
+```diff
+-static const struct TrainerMon sParty_Sawyer1[] = {
+-    {
+-    .lvl = 21,
+-    .species = SPECIES_GEODUDE,
+-    }
+-};
+```
+
+#### [src/data/trainers.party](..src/data/trainers.party)
+```diff
++=== TRAINER_SAWYER_1 ===
++Name: SAWYER
++Class: Hiker
++Pic: Hiker
++Gender: Male
++Music: Hiker
++Double Battle: No
++AI: Check Bad Move / Try To Faint / Check Viability
++
```

Comment:
We have mugshots and starting status now. can they be added as a showcase?

Reply:
I don't think the migration script is the right place to showcase it, it just shows what changes you can expect for a generic entry. We should probably document the .party file format, but that's out of scope for this PR

Reply:
You could pop them in `test/battle/trainer_control.party`, that's usually where I point people. I don't think we have any tests of the trainer data, just the party data, but they'd be easy to write and would help guarantee that `trainerproc` is producing the correct data :)

Reply:
Totally fine by me, but out of scope for this particular PR :)